core/src/main/java/com/scalar/db/api/ReplicationAdmin.java

```diff
@@ -31,10 +30,7 @@ default void createReplicationTables(Map<String, String> options) throws Executi
    */
   default void createReplicationTables(boolean ifNotExist, Map<String, String> options)
       throws ExecutionException {
-    if (ifNotExist && replicationTablesExist()) {
-      return;
-    }
-    createReplicationTables(options);
+    throw new UnsupportedOperationException(CoreError.REPLICATION_NOT_ENABLED.buildMessage());
```

Comment:
Shouldn’t we revise the error message itself now that we’ve determined the license type?